scripts/data_processing/public/archetypes_over_time.R

```diff
@@ -0,0 +1,344 @@
+#### Create data for stacked bar chart of archetypes over time ####
+#### Jordan Klein
+
+#### 1) Setup ####
+#### Run GISAID metadata extracting script
+source("../PPI/auto_extract_gisaid_metadata.R")
+
+#### Load packages
+library(tidyverse) # data wrangling
+library(tibble) # data wrangling
+library(janitor) # column naming
+library(countrycode) # country c?des
+library(lubridate) # date times
+library(readxl) # excel import
+library(zoo) # calculate rolling averages
+library(R.utils) # R utilities
+library(stringr) # to parse strings in R
+library(dplyr) # data wrangling
+library(readr) # read_csv
+library(stringi)
+
+#### Clear environment
+rm(list = ls())
+gc()
+
+### Load country codes-world bank dictionary
+dictionary <- read_csv("../../../data/processed/gisaid_countries.csv")
```

Comment:
Could you send me this file? Unless this is generated elsewhere, if so, can you point me to that script?

Reply:
Found it. Can you send me a brief description of what each script in here does and what order to run them in, and what output they generate for which Flourish chart (with a link to the Flourish chart)?

Reply:
I created a readme [here](https://github.com/PandemicPreventionInstitute/NGS-Capacity-map/blob/main/scripts/data_processing/public/NGS_timeseries_README.md)